pom.xml

```diff
@@ -7,7 +7,7 @@
     <groupId>com.github.wovnio</groupId>
     <artifactId>wovnjava</artifactId>
     <name>wovnjava</name>
-    <version>2.0.0</version>
+    <version>2.0.1</version>
```

Comment:
Perhaps could use minor version?
```suggestion
    <version>2.1.0</version>
```

I think "sending a new parameter to an API" can be considered more as a feature than a bugfix.

Reply:
it's not really a public API from wovnjava's perspective

Reply:
Wordpress ([PR](https://github.com/WOVNio/wordpress_plugin/pull/124)) uses minor version though. I think we should change one or the other.

Reply:
I don't think that was strictly correct, but for consistency i'll just use minor then